fighters/dedede/src/acmd/specials.rs

```diff
@@ -110,7 +110,7 @@ unsafe extern "C" fn game_specialsstart(agent: &mut L2CAgentBase) {
         WorkModule::on_flag(boma, *FIGHTER_DEDEDE_STATUS_GORDO_THROW_FLAG_THROW);
     }
     frame(lua_state, 25.0);
-    FT_MOTION_RATE(agent, 35.0 / (65.0-26.0));    
+    FT_MOTION_RATE(agent, 31.0 / (65.0-26.0));    
```

Comment:
`FT_MOTION_RATE_RANGE(agent, 26.0, 65.0, 31.0)`

new standard is to use this function, rather than math. usage is `agent`, `start_frame`, `end_frame`, `duration`